index.html

```diff
@@ -0,0 +1,128 @@
+            <select class="input-class" name="" id="dropdown">
+                <option value="" disabled selected>Select Current role</option>
+                <option value="student">Student</option>
+                <option value="full-time-job">Full Time Job</option>
+                <option value="full-time-learner">Full Time Learner</option>
+                <option value="prefer-not-to-say">Prefer not to say</option>
+                <option value="other">Other</option>
+            </select><br>
+        </div>
+
+        <div class="info-div">
+            <label for="recommend-friend">Would you recommend freeCodeCamp to a friend?</label><br>
+            <label>
+                <input type="radio" name="user-recommend" id="" value="definitely">
+                Definitely
+            </label><br>
+            <label>
+                <input type="radio" name="user-recommend" id="" value="maybe">
+                Maybe
+            </label><br>
+            <label>
+                <input type="radio" name="user-recommend" id="" value="not-sure">
+                Not sure
+            </label><br>
+        </div>
+        <div class="info-div">
+            <label for="favorite-feature">What is your favorite feature of freeCodeCamp?</label><br>
+            <select class="input-class" name="mostlike" id="">
+                <option value="challenges">Challenges</option>
+                <option value="projects">Projects</option>
+                <option value="community">Community</option>
+                <option value="openSource">Open Source</option>
+            </select><br>
+        </div>
+
+        <div class="info-div">
+            <label for="">What would you like to see improved? (Check all that apply)</label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="front-end-projects">
+                Front-end projects
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="back-end-projects">
+                Back-end projects
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="data-visualization">
+                Data Visualization
+            </label> <br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="challenges">
+                Challenges
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="open-source-community">
+                Open Source Community
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="citter-help-rooms">
+                Gitter help rooms
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="videos">
+                Videos
+            </label><br>
+            <label for="">
```

Comment:
Avoid using multiple-line break tag 
Line break tag<br> shouldn’t be used to make gaps between elements, instead split the text into separate paragraphs.
![image](https://user-images.githubusercontent.com/112048040/220046573-9c97559d-9326-4323-a7b0-c2a1fde23fa5.png)

Instead, use the <div> tag
![image](https://user-images.githubusercontent.com/112048040/220047085-0b5d5d78-3394-49df-9d89-820432eee625.png)